requirements/app/cloud.txt

```diff
@@ -3,3 +3,4 @@ redis==4.1.0
 docker==5.0.3
 setuptools == 59.5.0
 s3fs==2022.1.0
+arrow
```

Comment:
Let's avoid this extra dependency. This isn't useful for most users.

Reply:
I've dropped the dependency on arrow in a follow up commit.